### ISSUE TITLE
Bug with Create's Backtanks

### ISSUE THREAD

**Issue by emorydeer2** [BeansGalaxy/Beans-Backpacks-2#163]
![image](https://github.com/BeansGalaxy/Beans-Backpacks-2/assets/107522398/223f0798-42f0-4bfe-a390-d949441004d5)
This message appears when shift-clicking Backtanks in the inventory, I'm assuming you use some automated process to detect backpacks & prevent doubling up, so maybe add a teeny little exemption for backtanks.
(Unless this is intentional, which makes physical sense, though isn't great for gameplay.)

**Comment by BeansGalaxy** [comment]
The mod used to disable equipping them together but it was super buggy and didn't work out so I disabled it. I must've not disabled it entirely though. Can you still equip the Backtank with a Backpack?

**Comment by emorydeer2** [comment]
Yup, if I click it manually, or spam shift-click a bit, it'll go into the chest slot as it would otherwise.
